src/game/client/components/menus_demo.cpp

```diff
@@ -21,9 +21,9 @@
 #include "menus.h"
 
 CMenus::CColumn CMenus::ms_aDemoCols[] = {
```

Comment:
There is a script that collects all strings to be localized for releases based on the Localize("x") pattern. Also add a comment here similar to

https://github.com/teeworlds/teeworlds/blob/723fec877a85880c6fb86f447b6ad5b9307d5c91/src/game/client/components/menus_browser.cpp#L27

Reply:
done